docs/usage/customization-guide.md

```diff
@@ -234,10 +234,12 @@ further be used in label rules specified in
 [`custom`](#custom-feature-source) feature source.
 
 > **NOTE:** Be careful when creating and/or updating hook or feature files
-> while NFD is running. In order to avoid race conditions you should write into
-> a temporary file (outside the `source.d` and `features.d` directories), and,
-> atomically create/update the original file by doing a filesystem move
-> operation.
+> while NFD is running. In order to avoid race conditions you should write
```

Comment:
We talk about feature files AND hooks here so we should then add the dot-file filter to `getFeaturesFromHooks()`, too